var/spack/repos/builtin/packages/boost/package.py

```diff
@@ -243,8 +247,12 @@ def determine_bootstrap_options(self, spec, with_libs, options):
                 # error: duplicate initialization of intel-linux with the following parameters:  # noqa
                 # error: version = <unspecified>
                 # error: previous initialization at ./user-config.jam:1
-                f.write("using {0} : : {1} ;\n".format(boost_toolset_id,
-                                                       spack_cxx))
+
+                # Specifies -std=c++11 flag when using XL, which is needed
```

Comment:
this comment is a bit off-placed since it's in a general `if not intel` branch. maybe place above?

Reply:
I changed comment to make it more broadly applicable. Does that look acceptable now?